Core/GraphQL/GraphQLFiles/sell_fragment_fulfilmentPoint.graphql

```diff
@@ -0,0 +1,103 @@
+fragment FragmentFulfilmentPoint on FulfilmentPoint {
+  id
+  externalId
+  reference
+  type
+  status
+  imageUrl
+  mapImageUrl
+  lat
+  long
+  waitTime
+  prepTime
+  position
+  translations {
+    language
+    title
+  }
+  seatForm {
+    id
+    reference
+    imageUrl
+    completionButtonUrl
+  completionButtonTitle
+    showCompletionDate
+    allowUpdate
+    refreshOnSuccess
+    validationIntegration
+    expiresIn
+    translations {
+      language
+      title
+      description
+      submitButtonTitle
+      skipButtonTitle
+      completionButtonTitle
+      completionTitle
+      completionDescription
+    }
+    fields {
+      type
+      key
+      validationRegex
+      required
+      sortId
+      autoFill {
+        type
+        field
+      }
+      translations {
+        title
+        language
+        description
+        completionDescription
+        skipButtonTitle
+        completionTitle
+        completionButtonTitle
+        submitButtonTitle
+      }
+      selectOptions {
+        title
+        value
+        iconUrl
+      }
+    }
+  }
+  categories{...FragmentFulfilmentPointCategory}
+  venue {
+		id
+    label
+    name
+    status
+    isDefault
+    description
+    imageUrl
+    mapImageUrl
+    geoLat
+    geoLong
+    geoLatNorthWest
+    geoLongNorthWest
+    geoLatSouthEast
+    geoLongSouthEast
+    city
+    externalId
+    createdAt
+    updatedAt
+    venueIconUrl
+  }
+  timeslots {
+    id
+    externalId
+    reference
+    startTime
+    endTime
+    createdAt
+    updatedAt
```

Comment:
Do we need `createdAt` and `updatedAt` in the timeslots? These are the only difference between fulfilmentPoint.timeslots and order.timeslot, if we don't need these two fields, then we can create a fragment `Timeslot` to reuse it.